ansys/dpf/core/examples/downloads.py

```diff
@@ -728,3 +728,44 @@ def download_piston_rod(should_upload: bool = True, server=None, return_local_pa
 
     """
     return _download_file("piston_rod", "piston_rod.rst", should_upload, server, return_local_path)
+
+
+def download_cycles_to_failure(should_upload: bool = True,
+                               server=None,
+                               return_local_path=False) -> str:
+    """Download an example result file from a cyclic analysis and
+    return the download path.
+    If the server is remote (or doesn't share the memory), the file is uploaded or made available
```

Comment:
If the server is remote (or does not share memory), the file is uploaded or made available on the server side.